Mathlib/CategoryTheory/Enriched/Basic.lean

```diff
@@ -47,14 +47,18 @@ Note that we do not assume that `V` is a concrete category,
 so there may not be an "honest" underlying category at all!
 -/
 class EnrichedCategory (C : Type u₁) where
+  /-- The morphisms of this category -/
   Hom : C → C → V
+  /-- The identity morphism of this catgeory -/
   id (X : C) : 𝟙_ V ⟶ Hom X X
+  /-- Composition of two morphisms in this category -/
   comp (X Y Z : C) : Hom X Y ⊗ Hom Y Z ⟶ Hom X Z
   id_comp (X Y : C) : (λ_ (Hom X Y)).inv ≫ id X ▷ _ ≫ comp X X Y = 𝟙 _ := by aesop_cat
   comp_id (X Y : C) : (ρ_ (Hom X Y)).inv ≫ _ ◁ id Y ≫ comp X Y Y = 𝟙 _ := by aesop_cat
   assoc (W X Y Z : C) : (α_ _ _ _).inv ≫ comp W X Y ▷ _ ≫ comp W Y Z =
     _ ◁ comp X Y Z ≫ comp W X Z := by aesop_cat
 
+/-- Use `X ⟶[V] Y` to obtain the `V` object of morphisms from `X` to `Y`. -/
```

Comment:
maybe inherit doc here instead of a new one

Reply:
Done.